src/app/epilepsy/pages/events/events.component.ts

```
@@ -35,7 +35,7 @@ export class EventsComponent implements OnInit, OnDestroy {
           this.loading = false;
           this.hasMore = evnentsPage.hasMore;
         },
-        error: (error) => (this.error = error),
+        error: (error) => (this.error = error)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)